### DISCUSSION TITLE
What is "Soar"?

### DISCUSSION THREAD

**Discussion by GravisZro** [DescentDevelopers/Descent3#395] [Q&A] [answered]
I stumbled upon `Descent3/soar.h` and `Descent3/soar_helpers.h` and was wondering what "Soar" is? Is it a defunct online gaming service or something?

**Comment by JeodC** [comment, marked as answer]
Defunct AI engine plugin: https://groups.csail.mit.edu/robotics-center/public_papers/vanLent99.pdf

Not used as far as I can tell. I even removed the dll and game runs fine.
